tests/unit/s2n_self_talk_broken_pipe_test.c

```diff
@@ -43,6 +43,7 @@ void mock_client(struct s2n_test_io_pair *io_pair)
     sleep(1);
 
     conn = s2n_connection_new(S2N_CLIENT);
+    TEST_DEBUG_PRINT("Client connected %ld\n", time(0));
     config = s2n_config_new();
```

Comment:
We don't generally leave generic DEBUG messages like this in the tests. Do you think these will be useful to future developers, or are they specific to the problems you were debugging?

Reply:
They don't print by default, but I'm happy to remove them, they aren't proc race safe anyway.